server/src/test/java/org/elasticsearch/action/admin/indices/close/TransportVerifyShardBeforeCloseActionTests.java

```diff
@@ -138,7 +138,7 @@ public static void afterClass() {
     private void executeOnPrimaryOrReplica() throws Throwable {
         final TaskId taskId = new TaskId("_node_id", randomNonNegativeLong());
         final TransportVerifyShardBeforeCloseAction.ShardRequest request =
-            new TransportVerifyShardBeforeCloseAction.ShardRequest(indexShard.shardId(), clusterBlock, taskId);
+            new TransportVerifyShardBeforeCloseAction.ShardRequest(indexShard.shardId(), clusterBlock, false, taskId);
```

Comment:
we'll need to test the phase 1 somehow, but that can be done as a follow up

Reply:
I've added a simple test in f3b0166

Reply:
Thanks!